sysutils/mtools/Portfile

```diff
@@ -12,8 +12,12 @@ long_description    Utilities to access MS-DOS disks from Unix without \
 homepage            http://mtools.linux.lu/
```

Comment:
the homepage redirects now to `https://www.gnu.org/software/mtools/`, can you please update that?